src/towncrier/_builder.py

```diff
@@ -362,6 +362,7 @@ def render_fragments(
     top_underline: str = "=",
     all_bullets: bool = False,
     render_title: bool = True,
+    md_header_level: int | None = 1,
```

Comment:
I hope that we can make this only `int` without having to handle the `None` case